kubectl-mayactl/cli/command/cstor/volume/volume_info.go

```diff
@@ -5,13 +5,14 @@ import (
 	"html/template"
 	"os"
 
-	//"strconv"
-
-	cstorv1 "github.com/openebs/api/pkg/apis/cstor/v1"
+	cstortypes "github.com/openebs/api/pkg/apis/types"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/vaniisgh/mayactl/client"
 	"github.com/vaniisgh/mayactl/kubectl-mayactl/cli/util"
```

Comment:
can we change the pkg name `kubectl-mayactl` to something else ?

Reply:
kubectl-openebs ?

Reply:
Sounds good to me , even we can rename repo to `openebsctl`, we can discuss over a community sync up call.